music/cmd/zone.go

```
@@ -253,37 +226,34 @@ var zoneStepFsmCmd = &cobra.Command{
-		zone := dns.Fqdn(Zonename)
-		if zone == "." {
-			log.Fatalf("ZoneStepFsm: zone not specified. Terminating.\n")
-		}
+		PrepArgs("zonename")
 
 		data := music.ZonePost{
 			Command: "list",
 		}
-		zr := SendZoneCommand(zone, data)
+		zr := SendZoneCommand(tdns.Globals.Zonename, data)
 		zm := zr.Zones
```

golangci-lint (1.62.2)

ineffectual assignment to zm

(ineffassign)